Scripts/InfoPage/AbstractInfoSubPage.cs

```diff
@@ -0,0 +1,39 @@
+﻿using UnityEngine;
+
+namespace DUCK.DebugMenu.InfoPage
+{
+	public class AbstractInfoSubPage : MonoBehaviour
+	{
+		private const int DEFAULT_CHILDREN = 2;
```

Comment:
I'd probably leave an empty line between const and member fields

Reply:
Done